.travis.yml

```
@@ -21,6 +22,7 @@ after_success:
   - pip install sphinx sphinx-rtd-theme
   - make -C docs clean html
   - touch docs/build/html/.nojekyll # create this file to prevent Github's Jekyll processing
+  - coveralls
```

Maybe move this into `after_success` since it isn't really a test?

```suggestion

after_success:
  - coveralls
```

---

Hello. I don't see test section here. It is in after_success section already.